cmd/fyne_demo/tutorials/container.go

```diff
@@ -76,6 +96,21 @@ func makeCenterLayout(_ fyne.Window) fyne.CanvasObject {
 	return container.NewCenter(middle)
 }
 
+func makeDocTabsTab(_ fyne.Window) fyne.CanvasObject {
+	tabs := container.NewDocTabs(
+		container.NewTabItem("Tab 1", widget.NewLabel("Content of tab 1")),
```

Comment:
How about "Doc 1" instead of "Tab 1"? helps to compare with AppTabs demo.

Reply:
Done